src/particles/particle_creation.hpp

```diff
@@ -283,7 +257,7 @@ template <> struct ParticleCreationTraits<ParticleType::StochasticStellarPop> {
 				const amrex::Real vz = state_arr(i, j, k, HydroSystem<problem_t>::x3Momentum_index) / cell_density;
 				const int nscalars = Physics_Traits<problem_t>::numPassiveScalars;
 				const amrex::Real t_ff = std::sqrt(3.0 * M_PI / (32.0 * C::Gconst * cell_density));
-				const amrex::Real particle_mass = cell_density * cell_volume * eps_ff * dt / t_ff;
+				const amrex::Real particle_mass = cell_density * cell_volume * eps_ff_ * dt / t_ff;
```

Comment:
This is incorrect: the particle mass should be cell_density * cell_volume * eps_star.

Reply:
Done.

Reply:
I don't think you addressed my comment. It looks like you just changed `eps_ff` to `eps_ff_`, which is the same thing. I think this line of code should read

`const amrex::Real particle_mass = cell_density * cell_volume * eps_star`